src/javaserver/Routes/RoutesRegistrar.java

```diff
@@ -0,0 +1,42 @@
+package javaserver.Routes;
+
+import java.util.*;
+
+public class RoutesRegistrar {
+
+
+    private static final RoutesRegistrar thisInstance = new RoutesRegistrar();
+    private Map<String, Route> routes;
+
+    public static RoutesRegistrar getInstance() {
```

Comment:
Nice job finding a way to decouple the cob spec routes from the server and even using a pattern to do so.

I'd be curious as to if there's a good way to remove the need for the Single Pattern, since it can potentially cause difficulty with testing. Let's plan on discussing this.